contracts/WakuRln.sol

```diff
@@ -57,7 +99,14 @@ contract WakuRln is Ownable, RlnBase {
         if (!isValidCommitment(idCommitment))
             revert InvalidIdCommitment(idCommitment);
         if (memberExists[idCommitment] == true) revert DuplicateIdCommitment();
-        if (idCommitmentIndex >= SET_SIZE) revert FullTree();
+    }
+
+    function _validateIndexExpiration(
+        uint256 idCommitment,
+        uint256 index
+    ) internal view {
+        if (index >= SET_SIZE) revert InvalidIdCommitmentIndex(idCommitment, index);
```

Comment:
probably want to have a check for the `idCommitment` too? where we check if the member exists in the tree?

Reply:
err, oops, just saw the diff and didn't realize, feel free to ignore!